README.md

```diff
@@ -3861,7 +3861,6 @@ Other Style Guides
   - [Latest ECMA spec](https://tc39.github.io/ecma262/)
   - [ExploringJS](https://exploringjs.com/)
   - [ES6 Compatibility Table](https://compat-table.github.io/compat-table/es6/)
-  - [Comprehensive Overview of ES6 Features](http://es6-features.org/)
```

Comment:
it was working 4 days ago. let's use https://web.archive.org/web/20240404212626/http://es6-features.org/ in the meantime while i wait for a reply https://twitter.com/ljharb/status/1777371029893173271